### PR TITLE
Issue 280 v3

### PR DESCRIPTION
**Pull request by shrotia**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #<gh-issue-id>

Test URLs:
- Original: https://www.sunstar.com/<path>
- Before: https://main--sunstar--hlxsites.hlx.live/<path>
- After: https://$branch--sunstar--hlxsites.hlx.live/<path>

- [ ] New Blocks introduced in this PR
      If yes, please provide details below
**Block Name** : For e.g. cards
- [ ] Documented in sidekick library

- [ ] New variations introduced in this PR
**Variation Name** :  For e.g. cards (grid)
- [ ] Documented in sidekick library

- [ ] New mixins introduced in this PR
**Mixin Name** :  For e.g. compact, white
- [ ] Documented in sidekick library

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -394,14 +394,8 @@ async function loadEager(doc) {
...
     await waitForLCP(LCP_BLOCKS, SKIP_FROM_LCP, MAX_LCP_CANDIDATE_BLOCKS);
-    try {
-      /* if desktop (proxy for fast connection) or fonts already loaded, load fonts.css */
-      if (window.innerWidth >= 900 || sessionStorage.getItem('fonts-loaded')) {
-        loadFonts();
-      }
-    } catch (e) {
-      // do nothing
-    }
+    /* lonading fonts.css eagerly to avoid LCS */
```
Comment (bosschaert) [inline scripts/scripts.js:397]: Nit typo in comment

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -161,6 +161,13 @@
   max-width: 1200px;
 }
 
+.columns.heading-divider h2,
+.columns.heading-divider h3,
+.columns.heading-divider h4 {
+  border-bottom: 1px solid #3c4650;
```
Comment (bosschaert) [inline blocks/columns/columns.css:167]: The color is available as a variable here: https://github.com/hlxsites/sunstar/blob/main/styles/styles.css#L59

**Comment by bosschaert** [comment]
As a general comment your PR description looks a bit messy and lacks a link to the issue that this fixes and also doesn't have  the correct testing URLs.

**Comment by shrotia** [comment]
Ah sorry this was not supposed to be a PR.

**Pull request closed without merging**